doc/data_structures.rst

```diff
@@ -230,36 +233,40 @@ a ``DataSet`` will be described.
      -
      - X
 
-Sending
--------
+Build and Send a DataSet
+------------------------
 
-When building a ``DataSet`` to be stored in the database,
-a user can add any combination of tensors and metadata.
-To add a tensor to the ``DataSet``, the user simply uses
-the ``DataSet.add_tensor()`` function defined in
-each language.  The ``DataSet.add_tensor()`` parameters are the same
-as ``Client.put_tensor()``, and as a result, details of the function
-signatures will not be reiterated here.
+When building a ``DataSet`` object in-memory,
+a user can group various combinations of tensors and metadata that
+constrain to the supported data types in the table above. To illustrate, 
+to include a tensor in a ``DataSet`` object, use the ``DataSet.add_tensor()``
+function in a supported language. The SmartRedis DataSet API functions 
+are available in C, C++, Python, and Fortran. The DataSet API or ``DataSet.add_tensor()`` function, 
+operates independently of the database and solely 
+maintains the dataset object **in-memory**. The actual interaction with the redis database, 
+where a snapshot of the DataSet object is sent and stored, is handled by the Client API.
```

Comment:
Change the last sentence to something like "Storing the dataset in the backend databse is done via the client API put_dataset() method: